routers/voice.py

```
@@ -0,0 +1,459 @@
+            raise HTTPException(status_code = 404, detail = "An error occurred: bot doesn\'t exist")
+
+    except HTTPException as e:
+        raise e
+    except Exception as e:
+        raise HTTPException(status_code = 500, detail = f"An error occurred: {str(e)}")
+    
+@jwt_token
+@x_app_key
+@x_super_team
+@voice_router.post("/batch")
+async def batch(request: Request, token: str = Form(...), session_id: str = Form(...), file: UploadFile = File(...)):
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)